kroxylicious-operator/src/test/java/io/kroxylicious/kubernetes/operator/KafkaServiceReconcilerIT.java

```diff
@@ -56,7 +56,7 @@ void stopOperator() {
     }
 
```

Comment:
I'd like to see an IT that exercises the transitions when TLS bearing secret/configmap is a) removed, b) added late.